cmd/flags/admin.go

```diff
@@ -75,7 +76,7 @@ func (s *AdminServer) setLogger(logger *zap.Logger) {
 func (s *AdminServer) AddFlags(flagSet *flag.FlagSet) {
 	flagSet.Int(healthCheckHTTPPort, 0, healthCheckHTTPPortWarning+" see --"+adminHTTPHostPort)
 	flagSet.Int(adminHTTPPort, 0, adminHTTPPortWarning+" see --"+adminHTTPHostPort)
-	flagSet.String(adminHTTPHostPort, s.adminHostPort, "The host:port (e.g. 127.0.0.1:5555 or :5555) for the admin server, including health check, /metrics, etc.")
+	flagSet.String(adminHTTPHostPort, s.adminHostPort, fmt.Sprintf("The host:port (e.g. 127.0.0.1:%s or :%s) for the admin server, including health check, /metrics, etc.", s.adminHostPort))
```

Comment:
Looks like it's missing the value for the second parameter:

```
      --admin.http.host-port string                     The host:port (e.g. 127.0.0.1::14269 or :%!s(MISSING)) for the admin server, including health check, /metrics, etc. (default ":14269")
```

Reply:
Of course it needs to be there. Sorry for bothering you with such stupid mistakes. I've run your command and checked all outputs for now. It seems that `s.adminHostPort` already contains a colon therefore I updated the sourcecode to reflect that changes.